packages/react/src/components/Autocomplete/Autocomplete.stories.mdx

```diff
@@ -0,0 +1,98 @@
+import {ArgsTable, Source, Story, Canvas, Meta} from '@storybook/addon-docs';
+import dedent from 'ts-dedent';
+import StoryConfig from '../../../.storybook/story-config.ts';
+import Autocomplete from './Autocomplete.tsx';
+import TextField from '../TextField';
+
+export const meta = {
+  component: Autocomplete,
+  title: StoryConfig.Autocomplete.hierarchy,
+};
+
+<Meta title={meta.title} component={meta.component} />
```

Comment:
<Meta title="Inputs/Autocomplete" component={Autocomplete} />

Use this convention instead.

Reply:
Fixed with https://github.com/wso2/oxygen-ui/pull/165/commits/9062c46dc012aed02810bee2868409c5ba806c26